packages/presets/near-operation-file/src/utils.ts

```diff
@@ -36,21 +36,19 @@ export function extractExternalFragmentsInUse(
   oldVisit(documentNode, {
     enter: {
       FragmentSpread: (node: FragmentSpreadNode) => {
-        if (!ignoreList.has(node.name.value)) {
-          if (
-            result[node.name.value] === undefined ||
-            (result[node.name.value] !== undefined && level < result[node.name.value])
-          ) {
-            result[node.name.value] = level;
-
-            if (fragmentNameToFile[node.name.value]) {
-              extractExternalFragmentsInUse(
-                fragmentNameToFile[node.name.value].node,
-                fragmentNameToFile,
-                result,
-                level + 1
-              );
-            }
+        if (
+          !ignoreList.has(node.name.value) &&
+          (result[node.name.value] === undefined || level < result[node.name.value])
```

Comment:
I replaced 
```ts
if (
  result[node.name.value] === undefined ||
  (result[node.name.value] !== undefined && level < result[node.name.value])
)
```

by

```ts
if (
  result[node.name.value] === undefined ||
  level < result[node.name.value]
)
```

since `result[node.name.value] !== undefined` condition is redundant as is always will be `true`